docs/refs/api/drivers/BusInOut.md

```diff
@@ -1,9 +1,13 @@
 #### BusInOut
 
-Use the BusInOut interface as a bidirectional bus that collects a number of [DigitalInOut](DigitalInOut.md) pins that can be read and written as one value.
+BusInOut is an abstraction that takes up to 16 [DigitalInOut](DigitalInOut.md) pins and makes them appear as though they are linearly memory mapped for ease of use. This abstraction is useful for reading or writing multiple pins at the same time. In general this abstraction can be used to make code less cluttered, clearer, and take less time to write. 
```

Comment:
- I don't like the word "abstraction." Pleas remove both times.
- Query: In the first sentence, what does "makes them appear as though they are linearly mapped for ease of use" mean? Are they linearly mapped? Or do they just appear to be? Who or what linearly mapped them?
- Please delete "ease of use." It's obvious that it makes it easier, right?
- In the last sentence, please delete "In general".
- Please change the last sentence to active voice. [Example: "You can use BusInOut to write clearer code faster."]

Reply:
I literally copied all of this from the original doc in developer.mbed.org/handbook/BusInOut . 

Please feel free to modify as you see fit.